.github/workflows/d.yml

```diff
@@ -0,0 +1,27 @@
+name: Run sanitycheck
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dlang-community/setup-dlang@v1
+
+    - name: 'Run sanitycheck'
+      run: |
+        git clone https://github.com/dlang-tour/core ../tour
```

Comment:
Why not `actions/checkout` ?

Reply:
hm this is a different repository though, using the checkout action with a different repository seems like it would be quite a hassle. (need to modify repository and checkout directory. By specifying `../tour` it's some relative directory we don't know about which I think is good enough.

Reply:
I'm not sure that's true, the action is pretty straightforward, it's an extra `with` clause.
And "some directory we don't know about" doesn't sound too good, what if we don't have write permission to it ?
Currently that's not the case, but if they ever decide to make anything above `${{ github.workspace }}` RO, it'll be.